client/js/utils.js

```diff
@@ -85,3 +86,7 @@ function requestIdleCallback(callback, timeout) {
 		callback();
 	}
 }
+
+function isUserlistVisible() {
+	return $(".chan.active .sidebar").first().css("right") === "0px";
```

Comment:
I know this isn't the most foolproof way of doing this, and it works. Does anyone have any objections/suggestions for this?